src/Ninja/G1CP/Content/Symbols/string.d

```diff
@@ -241,3 +241,17 @@ func int G1CP_FindStringConstArrIdxI(var int symbId, var string needle) {
 func int G1CP_FindStringConstArrIdx(var string name, var string needle) {
     return G1CP_FindStringConstArrIdxI(MEM_GetSymbolIndex(name), needle);
 };
+
+/*
+ * Replace element in string array constant
+ */
+func int G1CP_ConstReplaceStr(var string strConst, var string originalStr, var string correctedStr) {
```

Comment:
Not to get too nit-picky now, but - for consistency - the other functions of these low-level groups use `needle` and `replace` as parameter names (see [string functions](https://github.com/AmProsius/gothic-1-community-patch/blob/4ae9164a4d3a0d50eb027a09634f2410b6a34a9a/src/Ninja/G1CP/Content/Bytecode/string.d#L132)). That might spare some confusion down the line, because there is concept of a "corrected" string in this low-level function.

Reply:
Nit-picky is just alright!
I completely agree.